packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, oneOf, arrayOf } from 'prop-types';
+import {
+  Burmese,
+  Bengali,
+  EasternArabic,
+  WesternArabic,
+} from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadLink } from './item';
+
+const MostReadItemProps = {
+  columns: {
+    group0: 5,
+    group1: 5,
+    group2: 5,
+    group3: 5,
+    group4: 5,
+    group5: 5,
+  },
+};
+
+const MostReadRankProps = {
+  item: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 1,
+    group3: 1,
+    group4: 1,
+    group5: 1,
+  },
+};
+
+const MostReadLinkProps = {
+  item: true,
+  columns: {
+    group0: 4,
+    group1: 4,
+    group2: 4,
+    group3: 4,
+    group4: 4,
+    group5: 4,
+  },
+};
+
+const MostReadListProps = {
+  enableGelGutters: true,
+  enableGelMargins: true,
+  columns: {
+    group0: 1,
```

Comment:
@lokh01 looking at the designs, we should have six columns for `group0` and `group1`
![Screen Shot 2019-11-18 at 2 59 36 PM](https://user-images.githubusercontent.com/21374761/69050964-1d92b900-0a14-11ea-85c5-2008f50ed59c.png)

Reply:
```
const MostReadItemProps = {
  columns: {
    group0: 6,
    group1: 6,
    group2: 3,
    group3: 3,
    group4: 4,
    group5: 4,
  },
};

const MostReadRankProps = {
  item: true,
  columns: {
    group0: 1,
    group1: 1,
    group2: 1,
    group3: 1,
    group4: 1,
    group5: 1,
  },
};

const MostReadLinkProps = {
  item: true,
  columns: {
    group0: 5,
    group1: 5,
    group2: 2,
    group3: 2,
    group4: 3,
    group5: 3,
  },
};

const MostReadListProps = {
  enableGelGutters: true,
  enableGelMargins: true,
  columns: {
    group0: 6,
    group1: 6,
    group2: 6,
    group3: 6,
    group4: 8,
    group5: 20,
  },
};
```
should fix this.

Reply:
Partially resolved - I've committed the changes to the columns but we are now waiting on a UX review since the spacing between the rank and the link is a bit too far so we're debating on whether or not to double the number of columns in `MostReadListProps`

Reply:
Will be handled by https://github.com/bbc/psammead/issues/2659